Components/NavigationFooter/functions.php

```diff
@@ -16,3 +17,12 @@
 
     return $data;
 });
+
+Options::addTranslatable('NavigationFooter', [
+    [
+        'label' => 'Copyright Text',
+        'name' => 'copyrightText',
+        'type' => 'text',
+        'default_value' => 'Copyright © 2019. All rights reserved'
```

Comment:
This might be anything, doesn't need to be a copyright notice. I think on our website, I would rather replace it with a notice "Powered by bleech.de" and link our website. So I was thinking about a wysiwyg and a more neutral label and name.

Reply:
Wysiwyg felt weird to me because of then having the ability to throw headings and lists into here but I didn't think about links. I will update it!

Reply:
I will use the 'basic' toolbar, just remembered we have this